xrootd.spec

```diff
@@ -1,16 +1,21 @@
-### RPM external xrootd 4.5.0
+### RPM external xrootd 4.6.1.RC2
 ## INITENV +PATH LD_LIBRARY_PATH %i/lib64
-%define tag 6a5db61a4ef13d1e79da71af47fbb8a660cf8aac
-%define branch cms/v%{realversion}
-%define github_user cms-externals
+%define tag 10ba4d7adfc5c66f33bcc77cf38ca45f426cb041
+%define branch stable-4.6.x
+
+%define github_user xrootd
 Source: git+https://github.com/%github_user/xrootd.git?obj=%{branch}/%{tag}&export=%{n}-%{realversion}&output=/%{n}-%{realversion}.tgz
 
+Patch1: xrootd-add-version
```

Comment:
Note, that this patch will go away once we have final 4.6.1 (if was done as temporary thing in GCC 7.0.1 IBs).